### DISCUSSION TITLE
How can I produce a list of all of my tags?

### DISCUSSION THREAD

**Discussion by OldMuskteer** [standardnotes/forum#2741] [Suggestions]
**Describe the issue**
Can a list of Tags I have used be created?

**Additional context**
I want to produce a list of all the Tags I'm using in Standard Notes. Is it possible to do this?

**Comment by JaspalSuri** [comment]
Hi @OldMuskteer, I'm afraid that there currently isn't a way to export all of the tags that you have created into a single list. 

Currently the ZIP files containing decrypted backups, that were exported from the desktop and [web ](https://app.standardnotes.org) apps, contain a folder named **Tags** that contain a file per tag, though.

**Comment by OldMuskteer** [comment]
As I feared JaspalSuri. Thanks for the information.
